apps/docs/content/components/Image.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Image
+status: Experimental
+# figma: https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0
+source: https://github.com/primer/brand/blob/main/packages/react/src/Image/Image.tsx
+storybook: '/brand/storybook/?path=/docs/components-image--playground'
+description: Use Image to create both normal and responsive image elements.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {Image} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+```jsx live
+<Image src="https://source.unsplash.com/random" alt="Random Image" />
+```
+
+### Picture
+
+The `as` prop can be used to set the container of the image to be a picture component.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  as="picture"
+/>
+```
+
+### Aspect ratio
+
+The `aspectRatio` prop can be used to set the aspect ratio of the image. This is useful when the image is not the same aspect ratio as the container.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  aspectRatio="16:9"
+/>
+```
+
+### Height
+
+The `height` prop can be used to set the height of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
+  alt="Random Image"
+  height={200}
+  aspectRatio="16:9"
+/>
+```
+
+### Width
+
+The `width` prop can be used to set the width of the image. This can be used along side the `aspectRatio` prop to create a responsive image the same size as other images.
+
+```jsx live
+<Image
+  src="https://source.unsplash.com/random"
```

Comment:
I think we discussed this before but I would like to use a different color for this image because I can't see this on my monitor. So I will pick the next darker grey available.